commands/configure_director_test.go

```diff
@@ -117,6 +118,132 @@ var _ = Describe("ConfigureDirector", func() {
 			Expect(logger.PrintfArgsForCall(11)).To(Equal("finished configuring resource options for bosh tile"))
 		})
 
+		Context("when the --config flag is set", func() {
+			Context("when other flags are set", func() {
+				It("returns an error", func() {
+					err := command.Execute([]string{"--config", "test.yml", "--az-configuration", "{}"})
```

Comment:
I like this "table-driven" style test. We should do more of this sort of thing going forward.